openapi/openapi.yaml

```
@@ -1691,6 +1691,156 @@ paths:
         "401":
           description: On invalid secret.
 
+  /graph/sync-job/solver:
```

Maybe could be parametrized with an enum?

```suggestion
  /graph/sync-job/{type}:
```